client/admin/components/common/Navbar.vue

```diff
@@ -30,7 +33,12 @@ export default {
   props: {
     drawer: { type: Boolean, default: true }
   },
-  computed: mapState('auth', ['user']),
+  computed: {
+    ...mapState('auth', ['user']),
+    fullName() {
+      return this.user.firstName + ' ' + this.user.lastName;
```

Comment:
You can use [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here. 
```js
return `${this.user.firstName} ${this.user.lastName}`
```
Maybe even use [object destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring)
```js
const { firstName, lastName } = this.user;
return `${firstName} ${lastName}`;
```